libcaf_core/caf/abstract_actor.cpp

```diff
@@ -189,15 +189,15 @@ void abstract_actor::register_at_system() {
     return;
   setf(is_registered_flag);
   [[maybe_unused]] auto count = home_system().registry().inc_running();
-  CAF_LOG_DEBUG("actor" << id() << "increased running count to" << count);
+  log::core::debug("actor {} increased running count to {}", id(), count);
 }
 
 void abstract_actor::unregister_from_system() {
   if (!getf(is_registered_flag))
     return;
   unsetf(is_registered_flag);
   [[maybe_unused]] auto count = home_system().registry().dec_running();
-  CAF_LOG_DEBUG("actor" << id() << "decreased running count to" << count);
+  log::core::debug("actor {} decreased running count to {}", count);
```

Comment:
```suggestion
  log::system::debug("actor {} decreased running count to {}", count);
```

Reply:
I have added the following in the robot tests to ensure that the build is successful after these changes. [robot/logging/output.robot](https://github.com/actor-framework/actor-framework/pull/1738/commits/37b2a807f84b724bd46c530aecda5cde592e822c#diff-173a2923d29ecc6d8e5f62972ffedeb7f9e4bb71788871c923a68d6d742b734f)

Reply:
We don't care about these. Instead of changing the baseline, please add `caf.system` to the ignored components for the test: https://github.com/actor-framework/actor-framework/blob/master/robot/logging/base.cfg